packages/react-native-web/src/modules/usePressEvents/PressResponder.js

```diff
@@ -345,9 +321,6 @@ export default class PressResponder {
         if (!disabled && isValidKeyPress(event)) {
           if (this._touchState === NOT_RESPONDER) {
             start(event, false);
-            // Listen to 'keyup' on document to account for situations where
-            // focus is moved to another element during 'keydown'.
-            document.addEventListener('keyup', keyupHandler);
```

Comment:
I think if the focus is moved to another element, then the `onPress` shouldn't be triggered anymore on the blurred element. So, I attach back the key-up listener to the element.

Reply:
@necolas hi, sorry if it's disturbing you, curious to know your thoughts about the solution here to fix the double `onPress` event.

Reply:
This change will regress the issue that is mentioned in the comment. The responder system has to "end" the current responder whenever keyup happens, so using `onKeyUp` will not work.

Reply:
Ah, I see. Thanks for the info!

> Maybe you just want to check again for disabled === true in the handler logic

Yes, this will fix for the disabled case, but the issue happens when the button lost focus too.

I'm thinking of 2 other solutions:
1. Instead of checking whether it's disabled or not, we can check whether the element is the active element or not. We will store the element when `onKeyDown` is triggered.
```diff
onKeyDown: (event) => {
  const { key, target } = event;
  if (!disabled && isValidKeyPress(event)) {
    if (this._touchState === NOT_RESPONDER) {
+     this.responderElement = target;
...

const keyupHandler = (event: KeyboardEvent) => {
  if (this._touchState !== NOT_RESPONDER && isValidKeyPress(event)) {
    end(event);
    document.removeEventListener('keyup', keyupHandler);
    ...
    const isNativeInteractiveElement = ...
+   const isActiveElement = this.responderElement === target;

+   if (onPress != null && !isNativeInteractiveElement && isActiveElement) {
      onPress(event);
    }
  }
```

or

2. Check `isNativeInteractiveElement` value in `onKeyDown` instead of `keyUpHandler`.
```diff
onKeyDown: (event) => {
  const { key, target } = event;
  if (!disabled && isValidKeyPress(event)) {
    if (this._touchState === NOT_RESPONDER) {
+     this.isNativeInteractiveElement = role === 'link' || ...
...
const keyupHandler = (event: KeyboardEvent) => {
  if (this._touchState !== NOT_RESPONDER && isValidKeyPress(event)) {
    end(event);
    document.removeEventListener('keyup', keyupHandler);
    ...
-   const isNativeInteractiveElement = ...

+   if (onPress != null && !this.isNativeInteractiveElement) {
      onPress(event);
    }
  }
```

Let me know what you think, thanks!

Reply:
I don't know, maybe the first one. Up to you to do the testing and make sure your PR doesn't break existing expectations.